impress-org/givewp
# Refactor: change "Add Form" button link and styles

Resolves [GIVE-999]

## Description



This PR makes the "Add Form" button from the form list view create a new form using the visual form builder. It also updates the styles to keep the consistency with the WordPress secondary and primary buttons.

## Affects



The "Add Form" button from the form list view.

## Visuals



https://www.loom.com/share/caff9f3671eb4defab5b470e391ae55e?sid=e9fee7e0-7817-4af9-ae05-1d9228918d74

## Testing Instructions



1. Go to `Give > All Forms` and make sure the new styles are applied to the buttons;
2. Go to `User > Profile` and change the admin color scheme; 
3. Check if the new color schema got applied to the buttons;
4. Click on the "Add Form" button and make sure it's creating new forms using the Visual Form Builder.

## Pre-review Checklist






[GIVE-999]: https://stellarwp.atlassian.net/browse/GIVE-999?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
